turbo.json

```diff
@@ -76,6 +76,10 @@
       "outputs": [],
       "dependsOn": ["^build"]
     },
+    "test:ci": {
+      "outputs": [],
+      "dependsOn": ["test"]
```

Comment:
No `^` is intentional here.

Adding the `^` will cause it to run at the top level, which depends on build. No `^` only runs it on workspaces, and none depend on build (unless they do... lol)